### ISSUE TITLE
Unable to serialize the response

### ISSUE THREAD

**Issue by piotrkulpinski** [wp-sync-db/wp-sync-db#89]
Does anyone of you get such message:
`Failed attempting to unserialize the response from the remote server. Please contact support.`

I'm getting it while trying to connect to remote server.
Any ideas what could cause such error?

**Comment by rob-gordon** [comment]
Same

**Comment by derby** [comment]
Same here, any solution?

**Comment by piotrkulpinski** [comment]
Yes, decided it's not worth to waste time on this shitty clone and bought the official version and it works perfectly!

https://deliciousbrains.com/wp-migrate-db-pro/

**Comment by michalsen** [comment]
If you are pulling from a ssl site to a local, change the pull address from https to http. worked here.

**Comment by frontenddevguy** [comment]
I'm having this problem with SiteOrigin builder, only on button and call to action widgets (possibly because they contain urls). As mentioned above, pulling from live (https) to localhost (http) but the pull address always changes automatically to http anyway.

Clicking on that particular widget causes a white screen on local, and for the widget content to load indefinitely on staging. Never seen this happen before and been using WP Sync DB for many years

**Comment by fahrenheit451fra** [comment]
Same here, but it was just access to the database (the user was not admin) in infomaniak :
![SCR-20240122-pneu](https://github.com/wp-sync-db/wp-sync-db/assets/45125765/4ebbfd43-4676-4a4e-af0c-9beb99106401)
I hope this helps some of you
